.github/workflows/release.yml

```diff
@@ -0,0 +1,34 @@
+name: Release
+
+on:
+  release:
+    types:
+      - published
+      - released
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      # - name: Checkout
+      #   uses: actions/checkout@v3
+      #   with:
+      #     fetch-depth: 0
+      # - name: Setup Node.js
+      #   uses: actions/setup-node@v3
+      #   with:
+      #     node-version: 16
+      #     cache: 'npm'
+      #     registry-url: 'https://repo.backbase.com/artifactory/api/npm/npm-backbase/'
+      #     scope: '@backbase'   
+      # - name: Release
+      #   uses: ./.github/actions/bump-version
+      #   with:
+      #     github_token: ${{ secrets.GITHUB_TOKEN }}
+      #     npm_token: ${{ secrets.NPM_TOKEN }}
+      - name: 'Extract project from tag'
+        id: set-project-from-tag
+        run: |
+          module="$(echo "$GITHUB_REF" | sed 's~^refs/tags/\(.*\)-v[0-9.]\+$~\1~')"
```

Comment:
I believe you can use `${{ github.event.release.tag_name }}` to fetch the tag name